microservices/submission-service/src/replay-validation/utils/readableToString.ts

```diff
@@ -0,0 +1,12 @@
+import type {Readable} from "stream";
```

Comment:
This might live in common?

Reply:
[082212b](https://github.com/SprocketBot/sprocket/pull/155/commits/082212b2a72f696fb4c7c92f5100fa4431050148)